user-guide/Advanced_Modules/Data_Aggregator/Data_Aggregator_settings.md

```diff
@@ -139,6 +139,9 @@ For example:
 }
 ```
 
+> [!NOTE]
+> The `PageSize` parameter within a query significantly influences the optimization of execution time and job performance. Adjusting this parameter can enhance efficiency. A lower PageSize necessitates more round trips but reduces the likelihood of timeouts when processing extensive rows. Conversely, a higher PageSize minimizes round trip durations but increases the potential for timeouts when requesting larger data volumes.
```

Comment:
I think we can you leave out this sentence? "Adjusting this parameter can enhance efficiency."
I would also nuance more like: "... can in some scenarios significantly influence execution times and overall job performance."
Because it doesn't always matter, and it doesn't affect the optimization of the query.